scene/gui/slider.cpp

```diff
@@ -255,8 +255,8 @@ void Slider::_notification(int p_what) {
 
 			Ref<StyleBox> style = theme_cache.slider_style;
 			Ref<Texture2D> tick = theme_cache.tick_icon;
-
-			bool highlighted = mouse_inside || has_focus();
+            
+            bool highlighted = editable && (mouse_inside || has_focus());
```

Comment:
```suggestion

			bool highlighted = editable && (mouse_inside || has_focus());
```

The indentation must be done with tabs.